dataprofiler/profilers/utils.py

```diff
@@ -328,14 +328,14 @@ def biased_skew(df_series: Series) -> float:
     M3 = 0 if np.abs(M3) < 1e-14 else M3
 
     if M2 == 0:
-        return 0.0
+        return np.float64(0.0)
```

Comment:
required for `assert_profile_equal` in testing ... otherwise could be a float64 & float